packages/packages/api/src/client/metriport.ts

```diff
@@ -41,6 +41,18 @@ export class Metriport {
     return resp.data.userId;
   }
 
+  /**
+   * For your given user ID, returns the user's connected providers
+   *
+   * @param {string} userId - The unique ID for the user in your app.
+   * @returns Array of connected providers
+   */
+  async getConnectedProviders(userId: string): Promise<string[]> {
+    const resp = await this.api.get<string[]>(`/user/${userId}/connect`);
```

Comment:
needs updating:
- `@returns Object containing array of connected providers.`
- `Promise<{connectedProviders: string[]}>` 
- `/user/${userId}/connected-providers`